src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -579,6 +597,22 @@ const submitEncryptModeForm: ControllerHandler<
     },
   })
 
+  if (responseMetadata)
+    statsdClient.distribution(
+      'formsg.submissions.normalisedReponseTime',
+      getNormalisedResponseTime(
+        responseMetadata.responseTimeMs,
```

Comment:
Should have flagged this out earlier, but maybe this should be `normResponseTimeMetadata` instead. The current name is kinda misleading haha

Reply:
Thank you! [820993d](https://github.com/opengovsg/FormSG/pull/6387/commits/820993df264f1d6f4b2e7ce3c8afca537c001eb2)